frontend/src/services/fetch/fetchers/searchFetcher.ts

```diff
@@ -74,22 +29,14 @@ export const searchForOpportunities = async (searchInputs: QueryParamData) => {
 export const downloadOpportunities = async (
   searchInputs: QueryParamData,
 ): Promise<ReadableStream<Uint8Array<ArrayBufferLike>>> => {
-  const { query } = searchInputs;
-  const filters = buildFilters(searchInputs);
-  const pagination = buildPagination(searchInputs);
+  const requestBody = formatSearchRequestBody(searchInputs);
```

Comment:
another simplification since we now have 3 fetch functions that need the same type of search request body format